wg-advisors/red-flags.md

```diff
@@ -20,6 +20,9 @@ not a policeman enforcing the rules.
 * Discussions are happening off-list.
 * Too much talk is happening on the private list.
 * Users' questions are going unanswered on the mailing list.
+* Users' reported issues are going uncommented and untouched.
```

Comment:
+1 on all of these. I think one of the big advisors roles might be to help to figure out how to deal with those oat different stages of the priojects (and here I see great opportunity to exchange learnings between projects - and advisors talking between themselves about some ideas to improve.

For me - for example it was eye-opening that those do not have to be committers to respond - and finding ways to encourage users/contributors to respond to each other and review each other's code - also as a way to become a committer.

Reply:
In PMCs like Maven we have many projects which are different and many times to do a PR review, comment issues  deeper knowledge is needed in a specific project ... but not all commiters good know all projects and so on ... 
So helping from contributors can be very valuable.

It can be very interesting how others projects deal with such cases ...

Reply:
Example from Airflow: We are using "collaborators" for that (aka known as triage team) where we ask and encourage people to become members of the team and help to triage the issues/PR, we actively "recruit" people to the team (often those people become committers or stop being active so it's natural but also  from time to time we have a "call for collaborators" - which reminds people that there is something like "collaborators" and why.

But it's far from perfect I would love to know how others are doing it :D